### ISSUE TITLE
Add an automated check that checks WCAG2AA compliance

### ISSUE THREAD

**Issue by daften** [StadGent/fractal_styleguide_gent-base#21]
To make sure atoms, molecules, ... are WCAG2AA compliant.

**Comment by meirege** [comment]
@Jeroen005 Added a WCAG2AA compliance check method in the gulpfile. This suggests some serious color changes though...

**Comment by meirege** [comment]
Further implementation of the WCAG2AA guidelines should be discussed during the next standup.

**Comment by daften** [comment]
@meirege please let me know when this'll be discussed

**Comment by daften** [comment]
@meirege when closing issues (as won't fix), mention why in a comment.

**Comment by meirege** [comment]
Closing this issue. It will be handled through our JIRA board by the Digipolis web team.
